test/runner.py

```diff
@@ -2,6 +2,9 @@
 import os
 import sys
 
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
Could you avoid this hack by using relative imports? `import .docs ad tdo`, etc?